source/common/quic/envoy_quic_client_connection.cc

```diff
@@ -229,7 +230,9 @@ void EnvoyQuicClientConnection::onFileEvent(uint32_t events,
   if (connected() && (events & Event::FileReadyType::Read)) {
     Api::IoErrorPtr err = Network::Utility::readPacketsFromSocket(
         connection_socket.ioHandle(), *connection_socket.connectionInfoProvider().localAddress(),
-        *this, dispatcher_.timeSource(), /*prefer_gro=*/false, packets_dropped_);
+        *this, dispatcher_.timeSource(),
+        Runtime::runtimeFeatureEnabled("envoy.reloadable_features.prefer_udp_gro"),
```

Comment:
You may need to set the socket option 
via SocketOptionFactory::buildUdpGroOptions() on this connection.

Reply:
thanks for the tip!  done